sdk/storage/Azure.Storage.Blobs.Batching/src/DelayedResponse.cs

```diff
@@ -80,7 +80,10 @@ public void SetLiveResponse(Response live, bool throwOnFailure)
         // We directly forward the entire Response interface to LiveResponse
         #region forward Response members to Live
         /// <inheritdoc />
-        public override int Status => LiveResponse.Status;
+        public override int Status =>
```

Comment:
Not sure how discoverable this is going to be but we can try.

Reply:
I think between the exception and ToString, we're giving folks as much as help as we can without introducing new types.